.clang-format

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
Do we need this duplicated here? I'd prefer to keep the master on the /moveit repo and wget it here when needed

Reply:
No. Will remove it.